bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -59,25 +56,32 @@ Property,Airflow,,Airflow,,
 Property,App,,App,Application,Software program
 Property,Brightness,,Brightness,,
 Property,Channel,,Channel,,
+Property,Color,,Color,,
 Property,ColorTemperature,,Color Temperature,,
 Property,Current,,Current,,
 Property,Duration,,Duration,,
 Property,Energy,,Energy,,
 Property,Frequency,,Frequency,,
 Property,Gas,,Gas,,
 Property,Heating,,Heating,,
+Property,HomeAway,,Home Away,,
```

Comment:
```suggestion
Property,HomeOccupancy,,Home Occupancy,,
```

This can be a Switch (At home, vs not at home), or a Number Item (how many people are at home).
But then can't we just use the existing `Presence` ?

Reply:
https://github.com/openhab/openhab-addons/pull/18514#discussion_r2038079146